Azure-Samples/aks-store-demo
# Storefront 001

## Purpose
Adding server side processing to store-front to allow proper function in AKS (k8s)
* ...

## Does this introduce a breaking change?
```
[X] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
docker compose build --no-cache
docker compose up
```

* Test the code
Get everything up and running via docker compose, get to storefront (either localhost:8080 or 0.0.0.0:8080)
Ensure products load
Ensure items added to cart
Ensure order is processed
```
```

## What to Check
Same as what to test
* ...

## Other Information
